tests/playwright/src/model/pages/main-page.ts

```diff
@@ -75,14 +75,18 @@ export abstract class MainPage extends BasePage {
     return await this.page.getByRole('row').first().isVisible();
   }
 
+  async getAllTableRows(): Promise<Array<Locator>> {
```

Comment:
Why is this a promise when it getting a list of Locators? It can be simplified to return Locator[]

Reply:
The return type of an async method must be the global Promise<T> type

Reply:
where did you see that?